packages/p2p/src/components/my-profile/my-profile-content.jsx

```diff
@@ -49,33 +48,7 @@ const MyProfileContent = () => {
             </React.Fragment>
         );
     } else if (my_profile_store.active_tab === my_profile_tabs.BLOCKED_ADVERTISERS) {
-        return (
-            <React.Fragment>
-                <BlockUserModal
-                    advertiser_name={my_profile_store.selected_blocked_user.name}
-                    is_advertiser_blocked
-                    is_block_user_modal_open={general_store.is_block_user_modal_open}
-                    onCancel={() => general_store.setIsBlockUserModalOpen(false)}
-                    onSubmit={my_profile_store.onSubmit}
-                />
-                <DesktopWrapper>
-                    <BlockUserTable />
-                </DesktopWrapper>
-                <MobileWrapper>
-                    <MobileFullPageModal
-                        body_className='block-user__modal'
-                        height_offset='80px'
-                        is_flex
-                        is_modal_open
-                        page_header_className='buy-sell__modal-header'
-                        page_header_text={localize('Blocked advertisers')}
-                        pageHeaderReturnFn={() => my_profile_store.setActiveTab(my_profile_tabs.MY_STATS)}
-                    >
-                        <BlockUserTable />
-                    </MobileFullPageModal>
-                </MobileWrapper>
-            </React.Fragment>
-        );
+        return <BlockUser />;
```

Comment:
Review on the new project structure, I structured the search box and list of blocked users into a single component since it makes it cleaner at `my-profile-content` and delegates all responsibility of rendering the search box + list of blocked users into the `<BlockUser />` component. (This is similar to how `<PaymentMethods />` was structured)

Reply:
I think this looks good man, the structure and naming conventions are better